client/style.sass

```diff
@@ -1,21 +1,66 @@
+body
+	margin: 0
+	
 #root
-	background-color: cyan
-	height: 100px
+	background-color: #d4e6e3
+	height: 100%
 	width: 100%
 
-.test
-	background-color: purple
-	height: 200px
-	width: 50px
+#App
+	display: flex
+	flex-direction: row
+	width: 100%
+	height: 100%
+	background-color: #d4e6e3
+
+.UserButton
+	width: 15%
+
+.Main
+	width: 70%
```

Comment:
Styling nitpick: This doesn't actually work well on mobile, where you want to use the full screen space

Reply:
which part? the % width? Would you use vw instead?

Reply:
I would have mobile be like, 100% - 5px of margins on each side. 
Think how facebook does it. Mobile is full width, and then theres a max width on desktop with growing whitespace

Reply:
Would you be able to expand on which width is the problem? I have it set up to split the screen into three columns, UserButton (now logoButton), Main, and AddButtons. The two outsides are 15% of the app, while the middle is 70%.

Are you suggesting that the Main should be 100% of the screen on mobile? or #App should be switched to 100% - 5px?

Reply:
The widths in the header should be completely independent of content width imo.
You can set the header stuff to whatever you want, but you should have the content width be effectively 100% on mobile, which i think is `.Main`